.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+    "connector": {
+      "name": "local",
+      "options": {
+        "pattern": ["**", "!.git/**", "!node_modules/**"]
+      }
+    },
+    "extends": ["development"],
+    "formatters": ["stylish"],
+    "hints": [
+      "button-type",
+      "disown-opener",
+      "html-checker",
+      "meta-charset-utf-8",
+      "meta-viewport",
+      "no-inline-styles:error"
+    ]
+  }
```

Comment:
Great on the set up the linters files but I think you forget to install the backages as the previous code reveiwer asked so kindly install them, Kindly install the stylelint and hintrc 💯🚀


- If you scroll down a little bit you can find how to install the linters packages since the first command for set up and the second to fix the errors 💯💯
![image](https://user-images.githubusercontent.com/58553711/185010847-63e8a6ba-6926-48fb-b8b9-b19f5d7975c3.png)